docs/config-file-ort-yml.md

```diff
@@ -178,6 +180,8 @@ The list of available options for `reason` are defined in
 Package curations can be added if you want to correct metadata of third-party dependencies.
 
 The following example corrects the source-artifact URL of the package with the id `Maven:com.example:dummy:0.0.1`.
+This requires to enable `enableRepositoryPackageCurations` in `ort.conf` like it is being done in
```

Comment:
Change to: Note that this feature requires `enableRepositoryPackageCurations` to be enabled in the `ort.conf`, see [reference.conf](../model/src/main/resources/reference.conf).